repository-ydb-v2/BUILD

```diff
@@ -62,6 +62,7 @@ java_test_suite(
         "@java_contribs_stable//:tech_ydb_test_ydb_tests_common",
         "@java_contribs_stable//:tech_ydb_ydb_auth_api",
         "@java_contribs_stable//:tech_ydb_ydb_proto_api",
+        "@java_contribs_stable//:tech_ydb_ydb_sdk_common",
```

Comment:
Was this a missing dependency? Do we need to also fix anything in `pom.xml` for `repository-ydb-v2` module?

Reply:
I think I'm the only person how use bazel for YOJ (this change doesn't related to this PR)